### PR TITLE
docs: improve transition between light & dark mode

### PR DESCRIPTION
**Pull request by PeachScript**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
Page flickers when switching between light mode and dark mode

Before improve:
Kapture 2020-03-17 at 20 57 03

After improve:
Kapture 2020-03-17 at 20 53 08

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | --        |
| 🇨🇳 Chinese | --        |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by PeachScript** [comment]
Oh no, why dose mode toggle not working in preview site?

**Comment by PeachScript** [comment]
It looks like no response from all controls, include version select, icon style buttons & etc.

**Comment by ycjcl868** [comment]
> Oh no, why dose mode toggle not working in preview site?

This may be a network problem

File: site/theme/static/common.less [hunk 1]
```diff
@@ -220,3 +220,19 @@ a {
...
+
+// keep transition consistent to make mode toggle animation be more smooth
+#header,
+#search-box,
+.main-wrapper,
+.ant-menu,
+.main-container,
+.aside-container.ant-menu-inline > .ant-menu-item-group > .ant-menu-item-group-title::after,
+.toc {
+  transition: all 0.3s @ease-in-out-circ !important;
```
Comment (ycjcl868) [inline site/theme/static/common.less]: `!important` ？

Comment (PeachScript) [reply site/theme/static/common.less]: My mistake, forgot demo style override 🤦‍♂️

Comment (PeachScript) [reply site/theme/static/common.less]: > `!important` ？

My mistake, forgot demo style override 🤦‍♂️

**Comment by ycjcl868** [comment]
lint error

**Review by ycjcl868 — APPROVED** [review]

**Pull request merged**